1600895432182	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1600906193326	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1600906193326	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1600906193328	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1600906193331	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1600906197866	Sync.Service	DEBUG	User-Agent: Firefox/80.0.1 (Linux x86_64) FxSync/1.82.0.20200904083306.desktop
1600906197866	Sync.Service	INFO	Starting sync at 2020-09-23 20:09:57 in browser session T2j2T3jk37pm
1600906197867	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1600906197870	Sync.Status	INFO	Resetting Status.
1600906197870	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1600906197870	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1600906197870	FirefoxAccounts	DEBUG	getOAuthToken enter
1600906197871	FirefoxAccounts	TRACE	getCachedToken returning cached token
1600906197871	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1600906197871	Sync.BrowserIDManager	INFO	Getting keys
1600906197871	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1600906197873	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1600906197873	Sync.BrowserIDManager	DEBUG	Getting a token using OAuth
1600906197873	FirefoxAccounts	DEBUG	getAccessToken enter
1600906197873	FirefoxAccounts	DEBUG	getOAuthToken enter
1600906197873	FirefoxAccounts	TRACE	getCachedToken returning cached token
1600906197873	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1600906197873	FirefoxAccounts	DEBUG	Using cached scopedKeys data for https://identity.mozilla.com/apps/oldsync
1600906197873	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1600906197874	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1600906197876	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1600906197876	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1600906197877	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1600906197878	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: TypeError: err.response is undefined(resource://services-sync/browserid_identity.js:508:13) JS Stack trace: _fetchTokenUsingOAuth/<@browserid_identity.js:508:13
1600906197878	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1600906197878	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1600906197878	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: TypeError: err.response is undefined(resource://services-sync/browserid_identity.js:508:13) JS Stack trace: _fetchTokenUsingOAuth/<@browserid_identity.js:508:13
1600906197880	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1600906197882	Sync.BulkKeyBundle	INFO	BulkKeyBundle being created for undefined
1600906197883	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/164549947/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1600906197883	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1600906197883	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1600906197884	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1600906197885	Sync.SyncScheduler	DEBUG	Next sync in 90000 ms. (why=reschedule)
1600906197885	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1600906197893	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
